README.md

```diff
@@ -1 +1,171 @@
-# Portfolio-setup
\ No newline at end of file
+<div align="center">
+
+  <img src="" alt="Techcolony" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio Mobile First Setup</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> This project is the portfolio mobile first setup, it is created using HTML and CSS with Linters also implemented. It has addition feature of responsivity to breaking points on various  mobile devices.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML5
+2. CSS3
+3. LINTERS
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] [OPTIONAL] please remove this placeholder texts as they are for your understanding only.